mat64/symmetric_test.go

```diff
@@ -76,6 +80,10 @@ func (s *S) TestSymAtSet(c *check.C) {
 	c.Check(t.At(1, 2), check.Equals, 12.0)
 }
 
+type addSymer interface {
```

Comment:
Since this is only used in the definition of method below, it can be defined there in the func.